packages/board/src/vaadin-board-row.js

```diff
@@ -255,6 +250,19 @@ class BoardRow extends ResizeMixin(ElementMixin(PolymerElement)) {
     }
   }
 
+  /** @private */
+  _shouldRecalculate(width, breakpoints) {
+    // Should not recalculate if row is invisible
+    if (this.offsetParent === null) {
```

Comment:
As @sissbruecker discovered in https://github.com/vaadin/web-components/pull/5792, the `offsetParent` property is also `null` when the element has a `position: fixed` ancestor, which means the layout will never be recalculated if the board is put in a dialog for example. I'd recommend using `isElementHidden` from `@vaadin/a11y-base/src/focus-utils.js` which takes this case into account.

Reply:
Addressed in the latest commit.